### ISSUE TITLE
Table Columns Reordering Not Working - Only header cells are getting moved, not the body of the table

### ISSUE THREAD

**Issue by khurambutt-REIsys** [swimlane/angular2-data-table#238]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
Column Reordering not working as expected, only the header cell is getting reordered when dragged but column in the datatable's body stays unmoved and doesn't adjust as per changes in the header

**Expected behavior**
Upon dragging table's header cell to reorder columns, both header's cell and its corresponding body's column should get reordered together as they are tied to each other.

**Reproduction of the problem**
Nothing special, for me just used the basic demo example from the repository and gives me the same problem in both Chrome and FF browsers(only ones I checked on). I used the version of angular2-data-table from both npm and the master branch(just before reporting this issue here) from this repository and caused the same problem. Started working on this Angular 2 project couple of weeks ago, used angular-cli to setup the project.

********    If you have a working example/demo somewhere please do direct me towards that or put it in the demo examples please ********

**Comment by amcdnl** [comment]
Can you reverify on 1.0?

**Comment by khurambutt-REIsys** [comment]
Sure, will give it a try.

**Comment by khurambutt-REIsys** [comment]
I checked it on both 1.0 and 1.1, no luck with either of them, in both cases it doesn't work. Moreover, while testing with these newer releases, found columns resizing to be not working anymore which was working better for me in the previous release.

I looked at demos here http://swimlane.github.io/angular2-data-table/ and I saw on couple of the demos where resizing is working and every other demo it doesn't work even though it says Resizeable is true by default as it is for orderable.
Working example from your demos: Virtual Scrolling with 100k Rows
Broken example from your demos: Basic - Auto Height

I appreciate you taking out the time to look at the problem, thanks a lot.

**Comment by amcdnl** [comment]
Please try one more time on 1.2 :)

**Comment by khurambutt-REIsys** [comment]
Sure thing, will do shortly.

Regards,
Khuram Butt
REI Systems ... Achieving Excellence Together
Office: 571.306.5218
khuram.butt@reisystems.com<mailto:%20khuram.butt@reisystems.com> | www.reisystems.com<http://www.reisystems.com/>

**Comment by khurambutt-REIsys** [comment]
I just tried with 1.2 but no luck what-so-ever, no change as far as I see. I don't see toggle columns example to be working correctly as well btw. 

For columns to be resizeable and orderable, there is nothing specific you have to do, correct? If there is, if you could please provide working examples too please, that will be great.

Thanks.

**Comment by amcdnl** [comment]
This seems to work fine for me in the demos, can you please provide a demo case of it not working?

**Comment by khurambutt-REIsys** [comment]
I checked it again on my local development environment, tried to use all your recent Releases tagged here and used this example https://github.com/swimlane/angular2-data-table/blob/master/demo/basic/basic-auto.ts...resize starts working only if I change the [columnMode] to "standard" otherwise it didn't work for me. But even after changing to standard mode, columns reordering didn't work for me at all.

Now coming to the demos here http://swimlane.github.io/angular2-data-table/, checked Basic demo(very first one) and columns resizing and reordering both not working; columns resizing works on Horz/Vert Scrolling demo though but not the columns reordering. I checked on Chrome( 54.0.2840.71 (64-bit)) and on columns reordering it throws an error as well on Console. So still doesn't work for me, which demo you are trying which works for you? I can try exactly that may be?

Demos for Angular1 are definitely working here http://swimlane.github.io/angular-data-table/ but not the Angular2 ones.

Thanks.
